src/ReinforcementLearningExperiments/deps/experiments/experiments/DQN/JuliaRL_NFQ_CartPole.jl

```diff
@@ -0,0 +1,95 @@
+# ---
+# title: JuliaRL\_NFQ\_CartPole
+# cover: assets/JuliaRL_BasicDQN_CartPole.png
+# description: NFQ applied to the cartpole environment
+# date: 2023-06
+# author: "[Lucas Bex](https://github.com/CasBex)"
+# ---
+
+#+ tangle=true
+using ReinforcementLearningCore, ReinforcementLearningBase, ReinforcementLearningZoo
+using ReinforcementLearningEnvironments
+using Flux
+using Flux: glorot_uniform
+
+using StableRNGs: StableRNG
+using Flux.Losses: mse
+
+function RLCore.Experiment(
+    ::Val{:JuliaRL},
+    ::Val{:NFQ},
+    ::Val{:CartPole},
+    seed = 123,
+)
+    rng = StableRNG(seed)
+    env = CartPoleEnv(; T=Float32, rng=rng)
+    ns, na = length(state(env)), length(first(action_space(env)))
+
+    agent = Agent(
+        policy=QBasedPolicy(
+            learner=NFQ(
+                action_space=action_space(env),
+                approximator=Approximator(
+                    model=Chain(
+                        Dense(ns+na, 5, σ; init=glorot_uniform(rng)),
+                        Dense(5, 5, σ; init=glorot_uniform(rng)),
+                        Dense(5, 1; init=glorot_uniform(rng)),
+                    ) |> gpu,
```

Comment:
```suggestion
                    ),
```
I dropped the gpu calls in #907 because it was only causing troubles and slowing down local testing, CI does not test on GPUs anyways.